### PR TITLE
added popup fucntionality

### PR DESCRIPTION
**Pull request by Hector096**
Milestone1: Design the Project Work Card
Milestone2: Add Js for functionality
Milestone3: Manual Test the functionality

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Hector096 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js [hunk 1]
```diff
@@ -13,3 +29,129 @@ function openMenu() {
...
+  
+             </div>
+  
+         </div>
+     </div>
+  </div>`;
+  workPopUpCard.innerHTML = popUpItem;
+}
+
+// eslint-disable-next-line no-unused-vars
```
Comment (mikenath223) [inline app.js:94]: - [x] Kindly remove all the eslint disabled rules here.

File: app.js [hunk 2]
```diff
@@ -13,3 +29,129 @@ function openMenu() {
   menu.style.backgroundColor = '#384af5';
   menu.style.display = 'block';
 }
+
+function popUpProject(popUpdata) {
+  const popUpItem = `<div id="popUp-view">
+     <div class="popUp-card">
+         <img src="images/close.png" alt="close" id="popUp-Close" onclick="popupClose();">
```
Comment (mikenath223) [inline app.js]: - [x] Please change this and all other regular functions to es6 arrow function syntax.

File: app.js (hunk 1 above)
Comment (Hector096) [reply app.js:94]: These are unused variables but used at run time. how am i suppose to clear eslint test?

File: app.js (hunk 2 above)
Comment (Hector096) [reply app.js]: Please can you elaborate a bit more on arrow functions as this is not a 1 line function?

**Pull request merged**